content/en/docs/concepts/configuration/even-pods-spread.md

```diff
@@ -210,50 +204,59 @@ If we apply "mypod.yaml" to this cluster, you will notice "mypod" stays in `Pend
 
 To overcome this situation, you can either increase the `maxSkew` or modify one of the constraints to use `whenUnsatisfiable: ScheduleAnyway`.
 
-### Example: Respecting NodeSelecor and NodeAffinity
+### Conventions
```

Comment:
I think this section should go before the examples.

Reply:
I'm ok either way. My original thought is that we should show some examples to give a general idea on this feature, and then explain some underlying conventions worthing noting.

Reply:
sg